.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.17.0
```

Comment:
Is this for asdf? If so, it seems the nodejs plugin already supports reading from .nvmrc which we have in this repo. See: https://github.com/asdf-vm/asdf-nodejs/
In which case, I don't think this file would be necessary. If you agree, could you please remove it from this PR? We'd just like to avoid duplicating the version numbers we need in multiple places. Thanks!

Reply:
I didn't know that... make sense to remove